erikiado/jp2_online
# Normalize 'Sacramentos' in Integrante

Before, sacramentos_faltantes was a field used to describe on plain text
the sacramentos that the integrante doesn't have. The problem it's that
we need this information for Indicadores.

For this reason, now the a new model Sacrament was created and a
Many-to-many relationship with Integrante was added.

#### What's this PR do?
This PR add the model 'Sacramento' and add a many-to-many relationship with Integrante. Also, the form to add and edit an integrante  shows a CheckboxSelectMultiple, to select the Sacramentos that the integrante has.
#### Where should the reviewer start?
First check the new Sacramento model in familias/test_models.py
Then, check the test migration added in familias/test_migrations.py, that evaluate that the four basics sacramentos (Bautismo, Comunión, Confirmación, Matrimonio).
Next, check the changes familias/forms.py on IntegranteForm, IntegranteModelForm.
Finally, evaluate the tests added on captura/tests.py


#### How should this be manually tested?
Log in as capturista, and create a new study, then the family dashboard, add a new Integrante, in the form, a section called "Sacramentos" has multiple checkboxes. Check some, and click save. After that, edit the integrante. The previously checked sacramentos should still being checked. You can edit the sacramentos.

#### Any background context you want to provide?
This change has been added due to the fact that we were need to normalize information to fulfill the directivos requests regarding the indicators.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
